client/src/components/NationwideMap.vue

```diff
@@ -345,6 +345,14 @@ export default defineComponent({
           this.marker.setLngLat([parseFloat(long), parseFloat(lat)]).addTo(this.map);
         }
         this.zoomToLongLat();
+      } else {
+        // Reset map to full view of U.S. when geo IDs are cleared.
+        if (this.map?.isStyleLoaded()) {
+          this.map?.jumpTo({
+            center: this.center,
+            zoom: 4,
```

Comment:
nit: Make zoom constant it's used twice in this file now

Reply:
Done